### PR TITLE
Issue #14 : reworked remove() method in classic & doubly linked lists

### PR DESCRIPTION
**Pull request by Fly-Style**
All remove() methods has been reworked in classic & doubly linked lists.

### CODE REVIEW

**Comment by Fly-Style** [comment]
[Tests are here](https://github.com/ShevaBrothers/easylib/pull/11/commits/303c31a1ad8718457bbb2d0b3079534cef9266a2)

File: src/core/base/test/ClassicListTest.kt [hunk 1]
```diff
@@ -40,12 +53,16 @@ class ClassicListTest {
...
 
     @Test fun removeFromClassicList() : Unit {
-        val len = 20
+        val len = 10
+        var checklist = mutableListOf<Int>(1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
         for (i in IntRange(1, len))
             list.add(i)
 
-        for (i in IntRange(1, len))
+        for (i in IntRange(1, len)) {
```
Comment (toniukan) [inline src/core/base/test/ClassicListTest.kt]: Do it in randomized order.

Comment (Fly-Style) [inline src/core/base/test/ClassicListTest.kt]: And how can I equals lists soon?

Comment (toniukan) [inline src/core/base/test/ClassicListTest.kt]: I mean remove numbers not 1, 2, 3..., but in some other order. (Right now it removes only first element each time).

Comment (Fly-Style) [inline src/core/base/test/ClassicListTest.kt]: Ah, okay.

**Pull request merged**